saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistrations.java

```diff
@@ -56,14 +58,15 @@ private RelyingPartyRegistrations() {
 	 * about the asserting party. Thus, you will need to remember to still populate
 	 * anything about the relying party, like any private keys the relying party will use
 	 * for signing AuthnRequests.
-	 * @param metadataLocation
+	 * @param metadataLocation This can be anything that the
```

Comment:
Let's please say something like "classpath- or file-based locations or HTTP endpoints" instead of tying the description to using `ResourceLoader`.